play.pokemonshowdown.com/src/battle-log.ts

```diff
@@ -141,7 +142,9 @@ export class BattleLog {
 			if (battle?.ignoreOpponent) {
 				if ('\u2605\u2606'.includes(rank) && toUserid(name) !== app.user.get('userid')) return;
 			}
-			if (window.app?.ignore?.[toUserid(name)] && ' +\u2605\u2606'.includes(rank)) return;
+			let ignore = PS.prefs.ignore || {};
```

Comment:
`PS` isn't guaranteed to exist here, unfortunately. For instance, Replays uses this code.

Reply:
You're going to need to redo the `window.PS?.prefs` thing, sadly.